src/wp-includes/bookmark-template.php

```diff
@@ -102,13 +102,24 @@ function _walk_bookmarks( $bookmarks, $args = '' ) {
 			$title = ' title="' . $title . '"';
 		}
 		$rel = $bookmark->link_rel;
-		if ( '' !== $rel ) {
-			$rel = ' rel="' . esc_attr( $rel ) . '"';
-		}
+
 		$target = $bookmark->link_target;
 		if ( '' !== $target ) {
+			if ( is_string( $rel ) && '' !== $rel ) {
+				if ( ! str_contains( $rels, 'noopener' ) ) {
```

Comment:
```suggestion
				if ( ! str_contains( $rel, 'noopener' ) ) {
```
🤦‍♀️ doh on my part.